### ISSUE TITLE
Images should always have alt text

### ISSUE THREAD

**Issue by emilypstewart** [j4502/Editor-Staff-Hub#10]
Ran your source on your [Bios page](https://github.com/j4502/Editor-Staff-Hub/blob/feature-bios/indexbios.html) through the validator and found a couple of small errors.

![Screenshot of W3C Validator errors](https://cloud.githubusercontent.com/assets/774386/11018173/a6610316-8581-11e5-9b1e-eb427987d570.png)

Good news: they're super easy fixes.

Just need to add alt text to some of the images. 

**Incorrect examples:**
`<img src="/assets/images/staff/myimage.png" width="200" alt="">`

 :-1: 

`<img src="/assets/images/staff/myimage.png" width="200" alt="photo">`

:unamused: 
`<img src="/assets/images/staff/myimage.png" width="200" alt=" ">`

:no_good:  
`<img src="/assets/images/staff/myimage.png" width="200">`

**Correct**
`<img src="/assets/images/staff/myimage.png" width="200" alt="portrait of R2D2">`

---
### What is alt text?

Alt text is used by screen readers (technology that helps people with visual impairments interact with your website). It uses the text to describe the image to a user relying on that technology. It should be concise, but still descriptive enough to give context.

**Example:**

![Elderly woman wearing headphones with look of disgust.](http://img.izismile.com/img/img5/20121119/640/the_most_awkward_stock_pics_part_4_640_28.jpg)

for this photo you could say `alt="photo of Elderly woman wearing headphones with look of disgust"` or something.
